content/ecosystem/_index.html

```diff
@@ -0,0 +1,444 @@
+---
+title: RAPIDS.ai | Ecosystem
+description: RAPIDS libraries, frameworks, extensions and more
+---
+
+  <!-- Hero Shortcode -->
+  {{< hero hero-text="Ecosystem" button-link="#software" button-text="Go to Software" >}}
+
+  <!-- Intro -->
+  <div id="hardware"></div>
+  <section class="section mb-6">
+    
+    <div class="container">
+      <h1 class="has-text-centered mb-5"> An ecosystem of hardware, <br> software, and developers</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-microchip"></i> Hardware </h2>
+          <p> RAPIDS runs on hardware powered by CUDA, the well-developed set of tools for GPU computing built by NVIDIA. RAPIDS works on PCs, workstations, data centers and the cloud. Learn more about the different types of hardware and find the best fit for your use case on NVIDA's product page for <a href="https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/"> Data Science Hardware <i class="fa-solid fa-arrow-up-right"></i></a> </p>
+         
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-code"></i> Software </h2>
+          <p> Set on a foundation of powerful hardware and parallelized by CUDA, the RAPIDS ecosystem is based on core projects, those extending its capabilities, and those built on top of RAPIDS.</p> <a href="#software">Jump To Section <i class="fa-solid fa-arrow-up-right"></i></a>
+
+          <br>
+          <img class="image img-center" src="/images/rapid-layers.png">
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-laptop-code"></i>  Developers </h2>
+          <p> We hope you will consider integrating RAPIDS into your solution or workflow. RAPIDS is for developers ranging from competent python users to C++ experts. Build something great with RAPIDS.</p> <a href="#developers">Jump To Section <i class="fa-solid fa-arrow-up-right"></i></a>
```

Comment:
This gives off a vibe of "RAPIDS is only for competent devs / experts" which is not super friendly. Let's reword.
```suggestion
          <p> We hope you will consider integrating RAPIDS into your solution or workflow. RAPIDS is built for Python and C++ developers alike. Build something great with RAPIDS.</p> <a href="#developers">Jump to Section <i class="fa-solid fa-arrow-up-right"></i></a>
```